src/Runner.Listener/JobDispatcher.cs

```diff
@@ -346,7 +346,38 @@ private async Task RunAsync(Pipelines.AgentJobRequestMessage message, string orc
                 }
 
                 var term = HostContext.GetService<ITerminal>();
-                term.WriteLine($"{DateTime.UtcNow:u}: Running job: {message.JobDisplayName}");
+
+                string workflowName = string.Empty;
+                if(message.Variables.TryGetValue("system.workflowFilePath", out var workflowPath))
+                {
+                    workflowName = workflowPath.Value.Split('/').LastOrDefault().Trim();
+                }
+
+                string repositoryName = string.Empty;
+                if(message.ContextData.TryGetValue("github", out var context))
+                {
+                    var githubContext = context as Pipelines.ContextData.DictionaryContextData;
+                    if(githubContext?.TryGetValue("repository", out var repository) == true)
+                    {
+                        repositoryName = (repository as Pipelines.ContextData.StringContextData)?.Value?.Trim();
+                    }
+                }
+                
+                string additionalInfo = string.Empty;
```

Comment:
should we just send [Job_Workflow_ref](https://cs.github.com/github/actions-dotnet/blob/b668258657a87d3fcc44a41a6e5b525323d51c7c/Actions/Runtime/Sdk/Server/TaskHub.cs#L5236) from service so you don't need any of these string magic?

Reply:
I agree that this is also an option, but I wanted the change to be exclusively on the runner.

Reply:
I can't access the link you sent, i get 404